src/sidebar/components/search-status-bar.js

```diff
@@ -41,13 +41,13 @@ function SearchStatusBar({ rootThread }) {
     filterQuery,
     focusModeFocused,
     focusModeUserPrettyName,
-    selectionMap,
+    hasSelectedAnnotations,
     selectedTab,
   } = useStore(store => ({
     filterQuery: store.getState().selection.filterQuery,
     focusModeFocused: store.focusModeFocused(),
     focusModeUserPrettyName: store.focusModeUserPrettyName(),
-    selectionMap: store.getSelectedAnnotationMap(),
+    hasSelectedAnnotations: store.hasSelectedAnnotations(),
```

Comment:
Use selector instead of grabbing state directly here...plus, what this consumer actually cares about is whether there are any selected annotations, not what they are.